Index: 1.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/1.js b/1.js
new file mode 100644
--- /dev/null	
+++ b/1.js	
@@ -0,0 +1,9 @@
+function add(a,b)
+{
+    const sum = a + b ;
+    return sum;
+}
+const result=add(2,5);
+console.log(`The sum of 2 and 5 is ${result}`);
+// The sum of 2 and 5 is 7
+
